src/viewer.c

```diff
@@ -401,27 +404,55 @@ static void zoom_image(const char* params)
     } else if (viewport_scale_def(&ctx.vp, params)) {
         name = params;
     } else {
+        struct str_slice param_slices[2];
```

Comment:
I would use short name, for example `ps` and `ps_num`, this will make the source code more readable.